src/components/home/elements/SendToken.vue

```diff
@@ -386,12 +387,12 @@ var initData = {
     amount: BigNumber(0),
     attachment: '',
```

Comment:
The attachment is not pass to JSON when call execute contract API. This will cause invalid signature error if there is attachment input.